BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -218,13 +218,17 @@ Rationale: Today, SSLv2 is considered insecure {{!RFC6176}}.
                <vspace blankLines='1'/>
 Rationale: SSLv3 {{?RFC6101}} was an improvement over SSLv2 and plugged some significant security holes but did not support strong cipher suites. SSLv3 does not support TLS extensions, some of which (e.g., renegotiation_info {{!RFC5746}}) are security-critical.  In addition, with the emergence of the POODLE attack {{POODLE}}, SSLv3 is now widely recognized as fundamentally insecure.  See {{DEP-SSLv3}} for further details.
 
-* Implementations SHOULD NOT negotiate TLS version 1.0 {{?RFC2246}}; the only exception is when no higher version is available in the negotiation.
+* Implementations MUST NOT negotiate TLS version 1.0 {{?RFC2246}}.
                <vspace blankLines='1'/>
-Rationale: TLS 1.0 (published in 1999) does not support many modern, strong cipher suites. In addition, TLS 1.0 lacks a per-record Initialization Vector (IV) for CBC-based cipher suites and does not warn against common padding errors.
+Rationale: TLS 1.0 (published in 1999) does not support many modern, strong cipher suites. In addition, TLS 1.0 lacks a per-record Initialization Vector (IV) for CBC-based cipher suites and does not warn against common padding errors. 
+               <vspace blankLines='1'/>
+NOTE: This recommendation has been changed from SHOULD NOT to MUST NOT on the assumption that {{?I-D.ietf-tls-oldversions-deprecate}} will be published as an RFC before this document.
```

Comment:
I think this citation needs to be normative ("!" within braces). We can change it in the future is that document is blocked for some reason.